lib/image_render.dart

```diff
@@ -225,13 +225,25 @@ double? _width(Map<String, String> attributes) {
   return widthString == null ? widthString as double? : double.tryParse(widthString);
 }
 
-double _aspectRatio(Map<String, String> attributes, AsyncSnapshot<Size> calculated) {
+double _aspectRatio(
+  Map<String, String> attributes,
+  AsyncSnapshot<Size> calculated,
+) {
+  double aspectRatio;
   final heightString = attributes["height"];
   final widthString = attributes["width"];
   if (heightString != null && widthString != null) {
     final height = double.tryParse(heightString);
     final width = double.tryParse(widthString);
-    return height == null || width == null ? calculated.data!.aspectRatio : width / height;
```

Comment:
I think you solution is _okay_ but would it have been enough to just check that `height` is not `0` and return `1` in that case?